automation/roles/confd/tasks/main.yml

```diff
@@ -83,7 +83,7 @@
         - /etc/confd/confd.toml
         - /etc/confd/conf.d/haproxy.toml
         - /etc/confd/templates/haproxy.tmpl
-      delegate_to: "{{ groups.balancers[0] }}"
+      when: "'balancers' in group_names"
```

Comment:
In this case, there will be an error because there will be a new haproxy node in the group that we are adding.
It is necessary to get files from the first node of the group.

Reply:
> what if the first node in the group will be a new node?

it is assumed that the user has read the [documentation](https://autobase.tech/docs/management/cluster-scaling)

> all necessary configuration files will be copied from the first server listed in the "balancers" group in the inventory file.

Reply:
> maybe this?

I think it will work.